package.json

```diff
@@ -9,6 +9,7 @@
   "files": [
     "dist",
     "docs",
+    "lib/tailwind",
     "src",
     "tailwind.config.js"
```

Comment:
i think the last one we need here is `tailwind.preset.js` since we require it in the config